cookbooks/aws-parallelcluster-install/resources/install_packages/install_packages_redhat8.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :install_packages, platform: 'redhat' do |node|
+  node['platform_version'].to_i == 8
+end
+unified_mode true
+default_action :setup
+
+use 'partial/_install_packages_rhel_amazon.rb'
+
+action :setup do
+  action_install_base_packages unless virtualized?
+  action_install_kernel_soruce unless virtualized?
```

Comment:
Do you have any idea how to solve it? I am afraid this will prevent us from performing a lot of tests on Docker, won't it?

Reply:
As we discussed we will tackle this in another PR

Reply:
Sure, I was just looking for ideas.